test/Interop/Cxx/enum/anonymous-with-swift-name.swift

```diff
@@ -0,0 +1,34 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-experimental-cxx-interop)
+
+// REQUIRES: executable_test
+
+import AnonymousWithSwiftName
+import StdlibUnittest
+
+var AnonymousEnumsTestSuite = TestSuite("Anonymous Enums With Swift Name")
+
+AnonymousEnumsTestSuite.test("SOME_OPTIONS") {
+  let red: SOColorMask = .red
+  let green: SOColorMask = .green
+  let blue: SOColorMask = .blue
+  let all: SOColorMask = .all
```

Comment:
To exercise several different name lookup paths, I'd recommend forming all of these with different syntaxes, like:

```
   let red: SOColorMask = .red
   let green = SOColorMask.green
   let blue = .blue as SOColorMask
   let all: SOColorMask = .all
```